docs/revert-links.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
```

Comment:
I think this file should be kept in the OR docs as I do not think is required here.